### PR TITLE
Refactor to use data engineering common

### PR DESCRIPTION
**Pull request by abbas123456**

### CODE REVIEW

File: app/commands/dev.py [hunk 1]
```diff
@@ -46,12 +45,22 @@ def db(create, drop, drop_tables, create_tables, recreate_tables):
             sqlalchemy_utils.create_database(db_url, encoding='utf8')
         if drop_tables or recreate_tables:
             click.echo('Drop DB tables')
-            app.db.drop_all()
+            drop_schemas()
         if create or create_tables or recreate_tables:
-            engine = create_engine(db_url)
-            create_schemas(engine)
             click.echo('Creating DB tables')
-            app.db.create_all()
```
Comment (marcuspp) [inline app/commands/dev.py:54]: This change results in the schemas being created without tables

Comment (abbas123456) [reply app/commands/dev.py:54]: Fixed

**Comment by marcuspp** [comment]
Javascript bundle seems to be missing
Screenshot 2020-05-12 at 12 28 52

So index.html styles do not see to be all loaded (comparing to what is currently live)

**Comment by abbas123456** [comment]
This is also a problem with the develop branch, will merge this PR and create a github issue.

**Pull request merged**